README.md

```diff
@@ -556,3 +560,171 @@ jobs:
           source_directory: source/directory/on/build/agent
           target: ${{ steps.sas.outputs.authorized_destination }}
 ```
+
+## setup_app
+
+### Description
+
+Configure git to use GitHub app in next steps of workflow. This strep is required for interation with GitHub api in the
```

Comment:
```suggestion
Configure git client in the workflow job to authenticate to GitHub using a GitHub App instead of builtin repo-scoped GITHUB_TOKEN.
```